src/Annotation/Desc.php

```diff
@@ -11,9 +11,12 @@
 
 namespace Translation\Extractor\Annotation;
 
+use JetBrains\PhpStorm\Deprecated;
```

Comment:
This is included in https://github.com/JetBrains/phpstorm-attributes, which is not referenced in composer.json, and we don't want too. 
`#[Deprecated]` is not yet a native PHP Attribute, so please use the Annotation instead

Reply:
Okido, will do. 

TBH  I was planning on removing the annotations all together at the finalized PR, since they shouldn't be used anymore. Not sure what you think about that?

Reply:
Done.